### PR TITLE
Added small image

### PR DESCRIPTION
**Pull request by MrKuBu**
Small image discord RPC

### CODE REVIEW

File: Source/DiscordUE4/Private/DiscordObject.cpp [hunk 1]
```diff
@@ -92,6 +92,21 @@ void UDiscordObject::SetLargeImage(const FString InKeyName)
 	}
 }
 
+void UDiscordObject::SetSmallImage(const FString InKeyName)
+{
+	activity.GetAssets().SetLargeImage(TCHAR_TO_UTF8(*InKeyName));
```
Comment (ryanjon2040) [inline Source/DiscordUE4/Private/DiscordObject.cpp:97]: Shouldn't this be Set Small Image?

File: Source/DiscordUE4/Private/DiscordObject.cpp [hunk 2]
```diff
@@ -92,6 +92,21 @@ void UDiscordObject::SetLargeImage(const FString InKeyName)
...
+	if (core)
+	{
+		core->ActivityManager().UpdateActivity(activity, [](discord::Result result)
+		{
+			uint8 ResultByte = (uint8)result;
+			DiscordObjectInstance->OnSmallImageSet.Broadcast(static_cast<EDiscordReturnResult>(ResultByte));
```
Comment (ryanjon2040) [inline Source/DiscordUE4/Private/DiscordObject.cpp:103]: OnSmallImageSer delegate is missing in header.

**Review by ryanjon2040 — CHANGES_REQUESTED** [review]
Thank you for the PR. I made some comments on the changes.

**Comment by MrKuBu** [comment]
ActivityAssets.LargeImage  - Large image
ActivityAssets.SmallImage  - Small image
ActivityAssets.LargeText  - Large image description (mouse pointing image)
ActivityAssets.SmallText  - Small image description (mouse pointing image)

**Comment by MrKuBu** [comment]
Can add sln file for build?

**Comment by ryanjon2040** [comment]
@MrKuBu I have merged the changes from other two PRs so I'm closing this one.

**Pull request closed without merging**